.ci/benchmarks.groovy

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env groovy
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'linux && immutable' }
+  environment {
+    REPO = 'apm-server'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    BRANCH_NAME_LOWER_CASE = "${env.BRANCH_NAME.toLowerCase().replaceAll('[^a-z0-9-]', '-')}"
+    AWS_ACCOUNT_SECRET = 'secret/observability-team/ci/elastic-observability-aws-account-auth'    
+    EC_KEY_SECRET = 'secret/observability-team/ci/elastic-cloud/observability-pro'
+    BENCHMARK_ES_SECRET = 'secret/apm-team/ci/benchmark-cloud'
+    TERRAFORM_VERSION = '1.1.9'
+    
+    CREATED_DATE = "${new Date().getTime()}"
+    JOB_GCS_BUCKET_STASH = 'apm-ci-temp'
+    JOB_GCS_CREDENTIALS = 'apm-ci-gcs-plugin'
+  }
+
+  options {
+    timeout(time: 3, unit: 'HOURS')
+    buildDiscarder(logRotator(numToKeepStr: '100', artifactNumToKeepStr: '30', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    rateLimitBuilds(throttle: [count: 60, durationName: 'hour', userBoost: true])    
+  }
+
+  stages {
+    stage('Checkout') {
+      options { skipDefaultCheckout() }
+      steps {
+        deleteDir()
+        gitCheckout(basedir: "${BASE_DIR}", shallow: false)
+      }
+    }
+
+    stage('Benchmarks') {
+      options { skipDefaultCheckout() }
+      environment {
+        // cloud tags
+        TF_VAR_BUILD_ID = "${env.BUILD_ID}"
+        TF_VAR_BRANCH_NAME = "${env.BRANCH_NAME}"
+        TF_VAR_ENVIRONMENT= 'ci'
+        TF_VAR_BRANCH = "${BRANCH_NAME_LOWER_CASE}"
+        TF_VAR_REPO = "${REPO}"
+        //profiles
+        ESS_SYSTEM_PROFILE = '1GBx1zone'
+        BENCHMARK_PROFILE = '64agents'
+        
+        GOBENCH_TAGS = "branch=${BRANCH_NAME_LOWER_CASE},benchmark_profile=${BENCHMARK_PROFILE},system_profile=${ESS_SYSTEM_PROFILE}"
```

Comment:
I'd say the branch attribute should match the branch name regardless of the naming convention. In addition, to include the:
* PR number, null if branch based. This is handy to look for PRs by ID, the branch attribute can also help with 
* target branch, null if branch based. This is handy to look for PRs that targeted a specific branch (8.2)

```suggestion
        GOBENCH_TAGS = "branch=${BRANCH_NAME},benchmark_profile=${BENCHMARK_PROFILE},system_profile=${ESS_SYSTEM_PROFILE},commit=${GIT_BASE_COMMIT},pr=${CHANGE_ID},target_branch=${CHANGE_TARGET}"
```

Those attributes are similar to the ones in https://github.com/elastic/apm-agent-nodejs/blob/b098c9fa6dbba76064e97e408e4ebd5f65140b99/test/benchmarks/utils/analyzer.js#L28-L37

Reply:
added the required tags